include/aspect/postprocess/visualization.h

```diff
@@ -383,6 +383,22 @@ namespace aspect
          * to be produced. Used to check for the next necessary output time.
          */
         double last_output_time;
+	
+        /**
+         * Maximum number of steps between the generation of graphical output. 
+	 * This parameter
```

Comment:
please align the comment lines. You can automate this process by using the `astyle` software (see https://github.com/geodynamics/aspect/blob/master/CONTRIBUTING.md#making-aspect-better). If you have the software installed, you can reformat all your changes by running `make indent` in the folder you compiled ASPECT in.